### ISSUE TITLE
get the name of the rigid bodies and markers

### ISSUE THREAD

**Issue by nag92** [qualisys/qualisys_cpp_sdk#43]
is it possible to get the name of the markers and rigid bodies using the SDK?

**Comment by OliverGlandberger** [comment]
**TL;DR - Yes.**

It is possible to get both the name of Rigid Bodies as well as the name of labeled trajectories using the RT Protocol (_see **QTM RT protocol.pdf**_). Although the syntax of course differs depending on which SDK you're using.

I took a quick look at [THIS](https://github.com/qualisys/qualisys_python_sdk/blob/master/examples/stream_6dof_example.py) example code in the Python SDK, and found the following:
`# Get 6dof settings from qtm`
`xml_string = await connection.get_parameters(parameters=["6d"])`
`body_index = create_body_index(xml_string)`

It's in `create_body_index` that the XML data retrieved from the RT Protocol is parsed.
